x/pairing/keeper/scores/pairing_score.go

```diff
@@ -0,0 +1,31 @@
+package scores
+
+import (
+	"reflect"
+
+	epochstoragetypes "github.com/lavanet/lava/x/epochstorage/types"
+)
+
+const (
+	// TODO: temp strategy weight until we implement taking it out of the policy
+	UNIFORM_WEIGHT uint64 = 1
+)
+
+// object to hold a provider's score and the requirements that construct the score (map[scoreReqName] -> score from that req)
```

Comment:
Let's stick to Go's doc style going forward (see: https://go.dev/blog/godoc)
For instance, here could be:
```
// PairingScore holds a provider's score with respect to a set of requirements (ScoreReq).
```

Reply:
done. see https://github.com/lavanet/lava/commit/13d50d5931010aab718c8591c5e649e16ad380c0